project/Dependencies.scala

```diff
@@ -98,7 +99,7 @@ object Dependencies {
 
   private val slf4jFacadeDependencies = List(
     "org.slf4j" % "slf4j-api" % slf4jV,
-    "com.typesafe.scala-logging" %% "scala-logging" % scalaLoggingV,
+    "com.typesafe.scala-logging" %% "scala-logging" % scalaLoggingV
```

Comment:
I believe the commas here and below were deliberately dangled to allow for more correct attribution in diffs when changes are made to these lists. I would recommend reverting all changes in this file apart from the required AWS changes.

Reply:
fixed